src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -809,7 +816,7 @@ EnsureTableCanBeColocatedWith(Oid relationId, char replicationModel,
  * and creates the schema of the given relationId. The function errors out if the
  * command cannot be executed in any of the worker nodes.
  */
-static void
+void
 EnsureSchemaExistsOnAllNodes(Oid relationId)
```

Comment:
Since it's now being used from other places, it might be nice to move this into a more appropriate file (e.g. commands/schema.c).

Reply:
Makes sense. Moved it to commands/schema.c